### ISSUE TITLE
Question about evaluation critetion in paper?

### ISSUE THREAD

**Issue by GostInShell** [theodeprelle/AtlasNetV2#10]
![image](https://user-images.githubusercontent.com/70795627/105932253-9dd89380-6087-11eb-8fe9-b2b8d1bd6829.png)
Here, it is said that the reconstruction task is evaluated by chamfer distance. But for surreal data, the ground-truth correspondences are known. Why not just compute the L2 distance for correponding points?

**Comment by theodeprelle** [comment]
Hi,

I believe we report the Chamfer Distance in every table of our paper. The reported loss is sometimes different from the training loss. In this case, we used the correspondences during the training and measured the quality of the reconstruction using the Chamfer Loss. 

Let me know if you have any other questions

**Comment by GostInShell** [comment]
Many thanks for the quick reply! I do have a few more questions.  By chamfer distance, is it the symmetric one computed as 

<img src="https://latex.codecogs.com/gif.latex?L_{cd}=\frac{1}{2}(\sum_{p}&space;\min_{q}\|&space;p-q\|^2_2&plus;\sum_{q}&space;\min_{p}\|&space;p-q\|^2_2)" title="L_{cd}=\frac{1}{2}(\sum_{p} \min_{q}\| p-q\|^2_2+\sum_{q} \min_{p}\| p-q\|^2_2)" /> 

By the way, I find that the results reported in the paper above conincide with the table below in the 3D-CODED respository. Do they also use the same loss?
![image](https://user-images.githubusercontent.com/70795627/106082815-5e2cad00-6156-11eb-87dd-2de012fd3db1.png)

**Comment by theodeprelle** [comment]
Hi, 

Yes this is the chamfer we used in the paper. You can find cuda code in the repo. And you're right. 3D-CODED and AtlasNet were a huge inspiration for us. We built our project using 3D-CODED and AtlasNet code.
